docs/Getting-Started/polykey-cli/installation.md

```diff
@@ -83,16 +93,18 @@ a terminal to restart it.
 - For ZSH:
 
 ```shell
-export PATH=~/Downloads:$PATH >> ~/.zshrc && source ~/.zshrc
+export PATH="$HOME/Downloads:$PATH"' >> ~/.zshrc && source ~/.zshrc
```

Comment:
There's a syntax violation here. You cannot use `"'`.

Double quote is necessary for interpolating variables. Single quote cannot be used. Remove the single quote here.

Reply:
Wait a minute I think I got confused here cause there's no `echo` command here.

See the bash script below. You're missing the echo.